sklearn/tree/export.py

```diff
@@ -72,7 +74,7 @@ def export_graphviz(decision_tree, out_file=SENTINEL, max_depth=None,
                     feature_names=None, class_names=None, label='all',
                     filled=False, leaves_parallel=False, impurity=True,
                     node_ids=False, proportion=False, rotate=False,
-                    rounded=False, special_characters=False):
+                    rounded=False, special_characters=False, decimals=3):
```

Comment:
Should it be `precision` or should we leave it as `decimals` to conform to the `assert_*` syntax?